locust/web.py

```diff
@@ -258,7 +273,8 @@ def reset_stats() -> str:
             if environment.runner is not None:
                 environment.runner.stats.reset_all()
                 environment.runner.exceptions = {}
-            return "ok"
+
+            return jsonify({"success": True, "message": "ok"})
```

Comment:
Are we unilaterally changing this return for non-modern-ui clients as well?

As much as the new return type is more sensible, it has the potential to break something, for someone.

We might consider conditionally altering this return response based on the UI being modern or not, for now at least.

Reply:
Yeah the old UI never consumes the response from this endpoint, so I wouldn't be worried about this being a breaking change. However I can add the conditional return, just to be extra safe 👍

Reply:
Yeah I do agree, I just know that [someone out there is calling that](https://www.hyrumslaw.com/) (for some reason) :D